add Edge version restricts. (#762)
MicrosoftEdge/MSEdgeExplainers: PwaDigitalGoods/explainer.md

## Usage Examples

The Digital Goods API allows the user agent to provide the above operations, alongside digital store integration via the Payment Request API. 

> **IMPORTANT**

> If you want to try this API with Microsoft Store, please use the latest Edge Beta/Dev/Canary. The Edge Stable currently does not support Digital Goods API on Windows.

Reviewer: Since this information will get out of date quickly, can you specify the exact version of Edge instead?

```suggestion
If you want to try this API with Microsoft Store, please use the Edge version X.X.X.X or greater.
```

Author: Modified.
